app/src/main/java/org/fossasia/openevent/data/Event.java

```diff
@@ -0,0 +1,94 @@
+package org.fossasia.openevent.data;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Created by MananWason on 27-05-2015.
+ */
+public class Event {
+    int id;
+    String name;
+    String logo;
+    SimpleDateFormat start;
+    SimpleDateFormat end;
+    float latitude;
+    float longitude;
+    String location_name;
```

Comment:
Follow `[Hungarian Notation][1]` everywhere, for json related purposes, where the json key maybe serialized with underscores, you can always put `@SerializedName("location_name")` as an annotation on top of the field.

[1]: http://en.wikipedia.org/wiki/Hungarian_notation